### PR TITLE
Replace primarySwatch with ColorScheme.fromSeed in the examples codelabs 

### PR DESCRIPTION
**Pull request by droidbg**
### Description

As a part of Issue: [#9713](https://github.com/flutter/website/issues/9713)
this PR updates and replace all instances of `primarySwatch` with `ColorScheme.fromSeed` in the examples/codelabs 

## Pre-launch Checklist

- [X] I read the [Effective Dart: Style] _recently_, and have followed its advice.
- [X] I signed the [CLA].
- [X] I updated/added relevant documentation (doc comments with `///`).
- [X] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-devrel channel on [Discord].

[Effective Dart: Style]: https://dart.dev/guides/language/effective-dart/style
[CLA]: https://cla.developers.google.com/
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: adaptive_app/step_03/lib/main.dart [hunk 1]
```diff
@@ -18,7 +18,7 @@ class MyApp extends StatelessWidget {
     return MaterialApp(
       title: 'Flutter Demo',
       theme: ThemeData(
-        primarySwatch: Colors.blue,
```
Comment (domesticmouse) [inline adaptive_app/step_03/lib/main.dart:21]: This directory captures the output of `flutter create`. Please revert this change.

**Review by domesticmouse — COMMENTED** [review]
This PR needs to be split into about three PRs, so I can gather appropriate sign offs.

**Comment by droidbg** [comment]
@domesticmouse Done with the changes.
These are the split PRs:
1. https://github.com/flutter/codelabs/pull/2135
2. https://github.com/flutter/codelabs/pull/2136
3. https://github.com/flutter/codelabs/pull/2137

**Review by domesticmouse — APPROVED** [review]
